src/js/MediaTrackInfo.js

```diff
@@ -68,7 +68,15 @@ export default class MediaTrackInfo extends React.Component {
             }
 
             if (this.props.paused) {
-                position = startPosition
+                // Clamp position to timer bounds
+                console.log('[!] Paused', Object.assign({}, this))
+                if(startDate < endDate){ //COUNTUP
```

Comment:
Might be slightly easier to take care of this in the previous `if` structure:

```
                startPosition = new Date(startDate.getTime() + offset)
                timeSince = new Date(startPosition.getTime() + ((now - this.props.updateTime) * this.props.countRate))
                // Start position is used if paused
                position = this.props.paused ? startPosition : timeSince
                // Clamp position to timer bounds
                position = endDate && position > endDate ? endDate : position
```

Then we could remove this section entirely

Reply:
`COUNTDOWN` version
```
                var startPosition = new Date(startDate.getTime() - offset)
                timeSince = new Date((now - this.props.updateTime) * this.props.countRate)
                var position = this.props.paused ? startPosition : new Date(startPosition - timeSince)
```